packages/files/src/main.rs

```diff
@@ -27,17 +27,45 @@ fn manifest() {
             "transforms": [
                 {
                     "from": "textfile",
-                    "to": ["html"],
+                    "to": ["html", "latex"],
                     "arguments": [],
                 },
                 {
                     "from": "image",
-                    "to": ["html"],
-                    "arguments": [],
+                    "to": ["html", "latex"],
+                    "arguments": [
+                        {
+                            "name": "caption",
+                            "default": "",
+                            "description": "The caption for the image"
+                        },
+                        {
+                            "name": "label",
+                            "default": "",
+                            "description": "The label to use for the image, to be able to refer to it from the document"
+                        },
+                        {
+                            "name": "type",
+                            "type": ["image", "svg"],
+                            "default": "image",
```

Comment:
I think we should infer the type based on the file extension, but we can leave that for later and open another issue

Reply:
This won't be necessary if we add the `[svg]` module I was talking about (I think.. are there other filetypes worth considering?)

You could also argue that it's nicer to skip the distinction and treat everything as an image, from the user's POV. Which do you think is better?

Reply:
Good point! I don't have a super strong opinion so I think it's your call : )